### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by htochko** [gama-platform/gama#40] [Q&A]
Seems I have problem with java lauching, suppose some env variables should be set to let Gama "see" jdk, need assistance.
I've downloaded full version with jdk for MacOS, but what I see is gray screen without interface. On running headlees I'm getting ae error "gama-headless.sh: line 24: ./../jdk/bin/java: No such file or directory" and it's trus - in ./../jdk is only "Contents" folder - may I should link somehow my java to that path, let me know what is wrong with my installation, any ideas?   
![Screenshot 2021-04-02 at 13 13 13](https://user-images.githubusercontent.com/1263851/113407675-74552480-93b6-11eb-9ba8-71c45a401e98.png)

**Comment by hqnghi88** [comment]
Hi,
More exact details about what did you have done will help us to guide you, which gama version did you use (which release)? with JDK embedded or not . If you are using the newest Mac os, here is a version which aim to resolve the problem https://github.com/gama-platform/gama/releases/tag/1.8.2
